src/openapi.yaml

```diff
@@ -4892,20 +4891,20 @@ components:
           type: string
           maxLength: 40
           minLength: 1
-        subAdresBuitenland_1:
```

Comment:
Since `vng-api-common==1.6.2` also upgrades `djangorestframework-camelcase` to `1.2.0`, several breaking changes are introduced

Reply:
I would recommend to pin drf-camelcase to an earlier version then, if this has such an impact. But it's up to @sergei-maertens if this is possible

Reply:
please check the changes on stable/1.0.x - there are workarounds for this: https://github.com/VNG-Realisatie/zaken-api/blob/stable/1.0.x/src/zrc/setup.py#L28